wms/src/components/ui/Modal.tsx

```
@@ -9,19 +15,39 @@ export default function Modal({
+            className="relative w-full max-w-md rounded-2xl bg-surface-raised p-6 shadow-xl"
+            initial={{ opacity: 0, scale: 0.95, y: 8 }}
+            animate={{ opacity: 1, scale: 1, y: 0 }}
+            exit={{ opacity: 0, scale: 0.97, y: 4 }}
+            transition={{
+              duration: 0.25,
+              ease: EASE_OUT_QUART,
+            }}
+          >
+            <div className="mb-4 flex items-center justify-between">
+              <h2 className="text-lg font-medium">{title}</h2>
+              <button onClick={onClose} className="text-sm text-neutral-500">
```

Biome (2.5.0)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)